units/systemd-ask-password-console.path

```diff
@@ -17,6 +17,7 @@ DefaultDependencies=no
 After=plymouth-start.service
 Before=paths.target cryptsetup.target
 Conflicts=emergency.service
+Before=emergency.service
 Conflicts=shutdown.target
 Before=shutdown.target
```

Comment:
why aren't the conflicts/before lines merged here?

Reply:
Because the second line is used for shutdown.target and initrd-switch-root.target, and I want to keep those two separate in a block that is the same in all units.